music21/stream/base.py

```diff
@@ -5371,14 +5371,13 @@ def toSoundingPitch(self, *, inPlace=False):
         'C3'
 
         If 'atSoundingPitch' is unknown for this Stream and all of its parent Streams
-        then will raise a StreamException:
+        then no transposition will take place (this used to raise an exception):
 
         >>> s = stream.Score()
         >>> p = stream.Part(id='partEmpty')
         >>> s.append(p)
         >>> p.toSoundingPitch()
-        Traceback (most recent call last):
-        music21.exceptions21.StreamException: atSoundingPitch is unknown: cannot transpose
+        <music21.stream.Part partEmpty>
```

Comment:
unfortunately, now it's unclear what this call is demonstrating.  

need to see what p's atSoundingPitch was before ('unknown') since that's never displayed, and what happens when 'unknown' stream calls toSoundingPitch() -- and that can no longer be demonstrated with an empty stream.

I'm -1 on the change to the status of 'unknown' streams because of this.

Reply:
I have modified the test a bit to show that p.atSoundingPitch is still 'unknown' after the call to toSoundingPitch.